app/boards/shields/launchpad/launchpad.keymap

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/ext_power.h>
+
+/* Layers */
+#define _FUNC 1
+#define _ADMIN 2
+
+/ {
```

Comment:
I'm not sure how feasible it is on such a tiny macropad, but finding a way to fit in `&bt BT_CLR` and maybe `&bt BT_NXT` and `&bt BT_PRV` might be useful.